modules/apps/oauth2-provider/oauth2-provider-scope-impl/src/main/java/com/liferay/oauth2/provider/scope/internal/liferay/ScopeDescriptorLocatorImpl.java

```diff
@@ -59,6 +82,7 @@ protected void activate(BundleContext bundleContext) {
 	@Deactivate
 	protected void deactivate() {
 		_scopedServiceTrackerMap.close();
+		_scopeDescriptorServiceTrackerMap.close();
```

Comment:
@brianchandotcom in these two lines the order is not important, that's why I removed the space between them. However I normally order them in the reverse order than they are initialized. Would you prefer by reverse initialization order and with space or without spaces and by alphabetical order?